src/tests/test_health.py

```diff
@@ -5,3 +5,10 @@
 def test_healthchecks(client):
     response = client.get("/status/health/")
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_runtime_error(client):
+    with pytest.raises(RuntimeError):
+        response = client.get("/500-test/")
```

Comment:
Of hij raised, of hij assert. Maar niet beide. je kan `api_client.raise_request_exception = False` instellen hiervoor.

Reply:
Ja, idd, die raise check is voldoende, ik haal de assert weg.